manifests/install/all-in-one.yaml

```diff
@@ -0,0 +1,83 @@
+# First part
+# Apply extra privileges to system:kube-scheduler.
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:kube-scheduler:plugins
+rules:
+- apiGroups: ["scheduling.sigs.k8s.io"]
+  resources: ["podgroups", "elasticquotas"]
+  verbs: ["get", "list", "watch", "create", "delete", "patch"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:kube-scheduler:plugins
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:kube-scheduler:plugins
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: system:kube-scheduler
+---
+# Second part
+# Install the controller image.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scheduler-plugins
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: scheduler-plugins-controller
+  namespace: scheduler-plugins
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: scheduler-plugins-controller
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["scheduling.sigs.k8s.io"]
+  resources: ["podgroups", "podgroups/status", "elasticquotas", "elasticquotas/status"]
```

Comment:
we have not enabled `status` subresource of podgroup, so `podgroups/status` is not necessary

Reply:
thanks. Could you help verify that when a PodGroup is created (without any matching pods), is its status displayed properly?

Reply:
> And if I create a deployment with 2 pods, the `phase` gets changed to `PreScheduling`, but if I delete the deployment, its phase stays in `PreScheduling` (I'd expect it to be updated back to `Pending`)

This may be a bug the status would not be back to `Pending`

Reply:
> This may be a bug the status would not be back to Pending

I will create an issue tracking this.

BTW: could you help confirm this issue:

> after I updated controller SA's rbac setting (replace Update with Patch), it shows properly. But it's not showing the number:

Reply:
In my test environment, as I showed https://github.com/kubernetes-sigs/scheduler-plugins/pull/163#discussion_r595711417,  we can see number.

The controller is deployed based on the manifests in this PR

Reply:
Created #166 to track the UX issues.